targets.js

```diff
@@ -1,8 +1,15 @@
 module.exports = [
-  '0.10.48',
-  '0.12.17',
-  '4.6.1',
-  '5.12.0',
-  '6.9.1',
-  '7.0.0'
+  {runtime: 'node', target: '0.10.48'},
+  {runtime: 'node', target: '0.12.17'},
+  {runtime: 'node', target: '4.6.1'},
+  {runtime: 'node', target: '5.12.0'},
+  {runtime: 'node', target: '6.9.1'},
+  {runtime: 'node', target: '7.0.0'},
+  {runtime: 'electron', target: '0.30.8'},
+  {runtime: 'electron', target: '0.32.3'},
+  {runtime: 'electron', target: '0.35.6'},
+  {runtime: 'electron', target: '1.0.2'},
+  {runtime: 'electron', target: '1.2.8'},
+  {runtime: 'electron', target: '1.3.12'},
+  {runtime: 'electron', target: '1.4.10'}
```

Comment:
Looking at the CI builds again there are some strange things happening for Electron<=0.35.0.

So it makes sense to only support >= 1.  Electron 0.36.0 to 1.0.2 have the same ABI so we will still be backwards compatible.
I'll change that 👍